src/_gettsim/sozialversicherung/rente/erwerbsminderung/anspruch.yaml

```
@@ -0,0 +1,15 @@
+---
+mindestpflichtbeitragsmonate:
```

Term correct? If you quickly find a reference, that would be great.

---

```suggestion
mindestpflichtbeitragszeiten_monate:
```
Just Pflichtbeitragszeiten in the law. Would be in line with what we do elsewhere.